internal/pkg/api/server/buffer/service/service.go

```diff
@@ -32,8 +37,44 @@ func (*service) HealthCheck(dependencies.ForPublicRequest) (res string, err erro
 	return "OK", nil
 }
 
-func (*service) CreateReceiver(dependencies.ForProjectRequest, *buffer.CreateReceiverPayload) (res *buffer.Receiver, err error) {
-	return nil, &NotImplementedError{}
+func (*service) CreateReceiver(d dependencies.ForProjectRequest, payload *buffer.CreateReceiverPayload) (res *buffer.Receiver, err error) {
+	ctx := d.RequestCtx()
+
+	store := d.ConfigStore()
+
+	config := model.Receiver{
+		ProjectID: d.ProjectID(),
+		Name:      payload.Name,
+	}
+
+	// Generate receiver ID from Name if needed
+	if payload.ReceiverID != nil {
+		config.ID = *payload.ReceiverID
+	} else {
+		config.ID = strcase.ToKebab(config.Name)
```

Comment:
Mame na to `strhelper.NormalizeName` (mozes tu funkciu premenovat na nieco lepsie).
Mali sme tam tiez na zaciatku `strcase` ale pribudali nam edge-cases, ked to negenerovalo pekny vystup.